frontend/src/components/OdhAppCard.tsx

```diff
@@ -187,12 +186,23 @@ const OdhAppCard: React.FC<OdhAppCardProps> = ({ odhApp }) => {
               {disabled ? disabledPopover : null}
               <Dropdown
                 onSelect={onOpenKebab}
-                toggle={<KebabToggle onToggle={(e, value: boolean) => onToggle(value)} />}
+                onOpenChange={(isOpened) => setIsOpen(isOpened)}
+                toggle={(toggleRef) => (
+                  <MenuToggle
+                    variant="plain"
+                    aria-label="Odh card toggle"
```

Comment:
`aria-label` should be a valid string we're comfortable displaying in the UI. 
Please use the old value which was `Actions`.